tests/test_database_types.py

```diff
@@ -179,11 +178,20 @@ def expand_params(testcase_func, param_num, param):
 
 
 def _insert_to_table(conn, table, values):
-    insertion_query = f"INSERT INTO {table} (id, col) VALUES "
-    for j, sample in values:
-        insertion_query += f"({j}, '{sample}'),"
-
-    conn.query(insertion_query[0:-1], None)
+    insertion_query = f"INSERT INTO {table} (id, col) "
+
+    if isinstance(conn, db.Oracle):
+        selects = []
+        for j, sample in values:
+            selects.append( f"SELECT {j}, timestamp '{sample}' FROM dual" )
+        insertion_query += ' UNION ALL '.join(selects)
```

Comment:
Wait, what? Oracle doesn't support values?

Reply:
I think in Oracle VALUES can only specify one row.

Reply:
😢 all good, Oracle is way more of an outlier than I thought